lib/modules/dosomething/dosomething_campaign/includes/CampaignTransformer.php

```diff
@@ -83,7 +83,7 @@ public function show($id) {
    * @return array
    */
   protected function transform($item) {
-    $data = array();
+    $data = [];
```

Comment:
Sorta. I've been trying to update and switch them out as often as I can based on what code I'm updating/refactoring ;)